src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -31,9 +39,13 @@ def test_self_cal(session):
     session.self_cal()
 
 
-def test_get_channel_name(session):
-    name = session.get_channel_name(1)
-    assert name == '0'
+def test_get_channel_names(multi_instrument_session):
+    # Once we have support for independent channels, we should update this test to include
```

Comment:
please add an issue to fix this and reference it from the code

Reply:
Done.